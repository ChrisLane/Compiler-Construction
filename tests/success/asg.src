// This is to test the asg expression. This should parse.
main() {
    x = 50;
    x = (10+20) / 3;
}
