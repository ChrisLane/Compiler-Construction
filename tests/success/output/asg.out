main (  ) { 
    Asg ( "x" := Const 50 ) ; 
    Asg ( "x" := Operator ( Divide, Operator ( Plus, Const 10 , Const 20 ) , Const 3 ) ) 
}
